tests/plots/test_text.py

```diff
@@ -29,3 +37,40 @@ def test_single_text_to_text():
         hierarchical_values=[test_hierarchical_values],
     )
     shap.plots.text(shap_values_test)
+
+
+def test_color():
```

Comment:
Would be great if you could use pytest-mpl and our test setup for plots here. Have a look at:
([baseline plots](https://github.com/shap/shap/tree/master/tests/plots/baseline), [pytest-mpl](https://www.bing.com/search?pc=MOZI&form=MOZLBR&q=pytest+mpl), [a test of ours](https://github.com/shap/shap/blob/master/tests/plots/test_beeswarm.py#L45))